aspnetcore/fundamentals/openapi/buildtime-openapi.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Generate OpenAPI documents at build time
+author: captainsafia
+description: Learn how to generate OpenAPI documents in your application's build step
+ms.author: safia
+monikerRange: '>= aspnetcore-6.0'
+ms.custom: mvc
+ms.date: 8/13/2024
+uid: fundamentals/openapi/buildtime-openapi
+---
+
+# Generate OpenAPI documents at build-time
```

Comment:
@Rick-Anderson @tdykstra This definitely needs some massaging for prose and formatting but the material covered is true to what I think we should cover. Feel free to modify this branch as you see fit.

Reply:
> @Rick-Anderson @tdykstra This definitely needs some massaging for prose and formatting but the material covered is true to what I think we should cover. Feel free to modify this branch as you see fit.

Can I merge  #33361 into this branch? You've reviewed  #33361 and @mikekistler  has approved my changes. My PR fixes the *toc.yml* merge conflict.  I think I can easily fix the other merge conflicts.